docs/developer-docs/web-apps/application-frontends/existing-frontend.mdx

```diff
@@ -1,129 +1,124 @@
 ---
-keywords: [intermediate, tutorial, frontend]
+keywords: [intermediate, tutorial, frontend, assets, asset canister]
 ---
 
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
-# Existing frontend applications
+
+# Using an existing frontend application
 
 <MarkdownChipRow labels={["Intermediate", "Tutorial"]} />
 
 ## Overview
 
-While numerous starter projects and examples exist for those who prefer to start from scratch, deploying an existing frontend application as a frontend canister is also a viable option.
+While numerous starter projects and examples exist for those who prefer to start from scratch, deploying an existing frontend application as a frontend <GlossaryTooltip>canister</GlossaryTooltip> is also a viable option for building an application on ICP.
 
-This guide provides an overview of how to deploy an existing frontend application built on the Next.js framework as a frontend canister.
+This guide provides an overview of how to deploy an existing Next.js application as a frontend canister.
 
 ## Next.js
 
 [Next.js](https://nextjs.org/) is an open-source web development framework providing React-based web applications with server-side rendering and static website generation.
 
-A full example exists on [this repository](https://github.com/jennifertrin/nextjsicp).
+[View the GitHub repo for this example](https://github.com/jennifertrin/nextjsicp).
 
 ### Prerequisites
 
-You’ll need the following to complete this guide:
-
--  [x] An existing Next.js application.
-
--  [x] Download [`dfx`](/docs/current/developer-docs/getting-started/install), the command-line execution environment that serves as the primary tool for creating, deploying, and managing dapps on the Internet Computer. You can reference how to download and configure dfx by reviewing the [install DFX section](/docs/current/developer-docs/getting-started/install/).
+Before you start, verify that you have:
 
+- [x] Downloaded and installed [`dfx`](https://github.com/dfinity/sdk/releases/latest).
 
-### Step 1: Create an application build
+- [x] An existing Next.js application. This guide will use the [Next.js ICP example](https://github.com/jennifertrin/nextjsicp).
 
-To start, you must generate and point the static files of a properly-built Next.js application for deployment on the Internet Computer.
+### Step 1: Build your Next.js application
```

Comment:
It may be worth noting that server methods, like getServerSideProps won't work since it's a client-only application

Reply:
Added note about this